lib/schedule.py

```diff
@@ -72,24 +76,24 @@ def get_time_point_infos_from_schedule_file(schedule_file_name: str) -> List[Tim
                 blue=int(line['blue_2']),
             )
 
-            time_point_infos.append(
-                TimePointInfo(
+            gradient_infos.append(
+                Gradient(
                     seconds=seconds,
                     color_1=color_1,
                     color_2=color_2,
-                    brightness=float(line['brightness']),
+                    brightness=float(line['brightness']) / 100.0,
                     scroll_speed=float(line['scroll_speed']),
                 )
             )
-        return sorted(time_point_infos, key=lambda time_point: time_point.seconds)
+        return sorted(gradient_infos, key=lambda gradient: gradient.seconds)
 
 
 SECONDS_IN_DAY = 24 * 60 * 60
 
 
 def get_colors_from_schedule_file(
```

Comment:
Maybe we update this to `get_scheduled_gradient`.

And we can pass in `seconds_into_day` as a kwarg. 

```
if seconds_in_day is None:
  seconds_in_day = get_seconds_into_day
```